src/flibustaApiHelper.ts

```diff
@@ -60,6 +63,7 @@ abstract class FlibustaAPIHelper {
   }
 
   public getFlibustaHTMLPage(url: string): Promise<HTMLElement | null> {
+    //
     return this.axiosController.instance().get<string>(url)
       .then((response) => parse(response.data).querySelector('#main'))
       .catch((error) => {
```

Comment:
`await` is removed because of `@typescript-eslint/return-await` rule, but await is used in another part of code: https://github.com/ynhhoJ/flibusta-api/blob/master/src/api/getBooksBySeries.ts#L44 

```typescript
  public async getBooksBySeries(name: string): Promise<Nullable<Array<BookSeries>>> {
    const booksListResult = await this.fetchBooksBySeriesFromFlibusta(name); // <------

    if (isNil(booksListResult)) {
      return undefined;
    }
}
```